packages/moonstone/GridListImageItem/GridListImageItem.less

```diff
@@ -4,6 +4,9 @@
 @import '../styles/skin.less';
 
 .gridListImageItem {
+	display:flex;
```

Comment:
I think you may want `display: inline-flex` here. Let me know if that works for your scenarios. Some of the alignment issues of this ticket initially were due to the component being used in an inline context. If it needs to be both, we may want to add an `inline` prop, like `Item`.

Maybe consider using `Layout` and `Cell`? It may offer some nice built-in benefits at low cost. It got a perf optimization recently.

Reply:
If applying `display: inline-flex`, it also worked. I had to apply `Layout` to `GridListImageItem`. I have done with `inline` and `orientation` props.